src/vl-map-search.js

```diff
@@ -88,12 +97,15 @@ export class VlMapSearch extends vlElement(HTMLElement) {
             return response.json();
           }).then((data) => {
             if (data && data.LocationResult) {
```

Comment:
Ken de specs niet van LocationResult, ik vind op https://loc.geopunt.be/ niet onmiddellijk een referentie, zijn we zeker dat LocationResult altijd BoundingBox bevat?

Reply:
Op https://loc.geopunt.be zie ik staan dat LocationResult een BoundingBox bevat, daar lijkt het in de praktijk ook op al wordt er geen duidelijke uitspraak gedaan over het al dan niet altijd aanwezig zijn.